docker/package/scripts/tezos-baker-start

```diff
@@ -45,3 +45,7 @@ if [[ -z "$BAKER_ADDRESS_ALIAS" ]]; then
 else
     launch_baker "$BAKER_ADDRESS_ALIAS" "$@"
 fi
+
+# Waiting till node start to respond
+
+while ! "$tezos_client" --endpoint "$NODE_RPC_ENDPOINT" rpc list &> /dev/null; do sleep 1; done
```

Comment:
Hmmm, does it make sense to run this loop after the `launch_baker` is invoked? This way baker process still may die in the background due to connection error

Reply:
Makes no sense actually, fixed